docs/docs/destinations/README.md

```diff
@@ -23,16 +23,16 @@ The docs below discuss features common to all Destinations. See the [docs for a
 
 ### Using destinations in workflows
 
-You can send data to Destinations in [Node.js code steps](/workflows/steps/code/), too, using `$send` functions.
+You can send data to Destinations in [Node.js code steps](/workflows/steps/code/), too, using `$.send` functions.
```

Comment:
`/workflows/steps/code/` 404s since you renamed the file.

Reply:
I didn't comment in the other docs, but we'll need to modify the links there, too.

Reply:
Yes if that works feel free. You may also need to add equivalent redirects in `vercel.json` so we point Google to the new link.